### PR TITLE
add SQLite3Connection>>#execute:withAll:

### PR DESCRIPTION
**Pull request by vonbecmann**
fix: #105

### CODE REVIEW

File: src/SQLite3-Core/SQLite3Connection.class.st [hunk 1]
```diff
@@ -47,3 +47,13 @@ SQLite3Connection >> execute: anSQLText with: aCollection doing: aBlock [
...
 			ensure: [ cursor finalizeStatement ]
 ]
+
+{ #category : #'public API - operating' }
+SQLite3Connection >> execute: anSQLText withAll: parameterCollection [
+	"execute a sql statement with many parameters"
+
+	| stmt |
+	stmt := self prepare: anSQLText.
+	parameterCollection do: [ :parameters | stmt execute: parameters ].
```
Comment (vonbecmann) [inline src/SQLite3-Core/SQLite3Connection.class.st:57]: should use 
```
[ parameterCollection do: [ :parameters | stmt execute: parameters ] ] ensure: [ stmt finalize ]
```
??

**Comment by astares** [comment]
Code not covered by test

**Comment by vonbecmann** [comment]
see SQLite3ConnectionTest >> testExecuteWithAllInsertManyValues

**Comment by vonbecmann** [comment]
added another testcase that update many values.

**Pull request merged**